docs/marketplace/user/intro-to-spryker-marketplace/merchant-onboarding.md

```diff
@@ -0,0 +1,146 @@
+5. The operator decides what kind of payment process to implement—for example, whether or not to use a *Payment Service Provider (PSP)*.
+6. An order management process for fulfillment, shipping, and returns is established for merchants through Spryker State Machine.
+7. A final check is conducted on the merchant's public profile, products, and offer quality. The operator activates merchants and their products and offers in Back Office. Everything can be activated via a data importer as well.
+
+Each step is described in the following sections.
+
+### 1) Merchant: Put in a request to join
+
+The official onboarding starts with a merchant requesting to join the marketplace. 
+
+To request a new merchant account, you can build a landing page with Spryker CMS where potential merchants can find the email address or a form to contact the operator to join the marketplace. In some cases, the operator may contact their partners and inform them about the marketplace.
+
+### 2) Operator: Verify (the KYC process) and register a merchant
+
+1. Gather data a merchant provides, such as a registration number and the company behind the merchant, including representatives' information and their IDs.
+2. To identify the trust level and the need for additional information, validate your merchant through public registers. 
+
+{% info_block infoBox %}
+
+To accelerate the validation process, you can use relevant Spryker technology partners. For more details, reach out to your Spryker contact.
+
+{% endinfo_block %}
+
+The official relationship between the operator and merchant can be established by providing a contract to a merchant. For specific merchants, custom contracts can be used. Otherwise, boilerplate agreements must be available. Contracts can be made as part of terms and conditions that must be accepted prior, together with platform rules, customer communication guidelines, data policy, and SLA agreement. Clarification is also required regarding who the product data owner is, in which categories the merchant is allowed to sell, and the commissions and fees the merchant must pay the operator.
+
+![edit-merchant](https://spryker.s3.eu-central-1.amazonaws.com/docs/marketplace/user/intro-to-spryker-marketplace/merchant-onboarding/step-2-edit-merchant.png)
+
+### 3) Operator: Approve the merchant
+
+1. In the Back Office, navigate to **Marketplace&nbsp;<span aria-label="and then">></span> Merchants**.
+2. On the **Overview of Merchants** page that opens, in the **Actions** column, click **Approve Access**. 
+
+By approving the merchant, the operator enables merchant users to access the Merchant Portal to complete their public-facing profile and create products and offers. 
+A merchant admin user needs to be created by the operator based on data provided through the registration form, such as the email or first and last name of the contact person. 
```

Comment:
how do they create the merchant admin user? We have some information on that [here](https://docs.spryker.com/docs/pbc/all/merchant-management/202212.0/marketplace/manage-in-the-back-office/manage-merchants.html#reference-information-creating-merchants), but nothing specific

Reply:
@mikost7 this is question for you

Reply:
It is done in Back Office in Merchant > Users tab. the permissions are then managed on the main Users pages. this one we can describe better actually.

Reply:
@mikost7 @lenadoc I suggest the following update:
```suggestion
Then, the operator needs to create a merchant admin user based on data provided through the registration form, such as the email or first and last name of the contact person. For this, the operator finds a merchant user in the Back Office, in **Merchant&nbsp;<span aria-label="and then">></span> Users** and assigns them the required permissions.
```